{
    "test_case": "second_test",
    "result": "code error",
    "steps": [
        {
            "message": "Error",
            "screenshot": "Error_8e3d1.png",
            "error": {
                "message": "TypeError: Object of type TextIOWrapper is not JSON serializable",
                "description": "Traceback (most recent call last):\n  File \"c:\\users\\alexandr.may\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\golem\\test_runner\\test_runner.py\", line 173, in run_test\n    self.test_module.test(execution.data)\n  File \"C:\\Users\\alexandr.may\\testdir\\projects\\some_project\\tests\\second_test.py\", line 32, in test\n    json.dump(file, data)\n  File \"c:\\users\\alexandr.may\\appdata\\local\\programs\\python\\python37-32\\lib\\json\\__init__.py\", line 179, in dump\n    for chunk in iterable:\n  File \"c:\\users\\alexandr.may\\appdata\\local\\programs\\python\\python37-32\\lib\\json\\encoder.py\", line 438, in _iterencode\n    o = _default(o)\n  File \"c:\\users\\alexandr.may\\appdata\\local\\programs\\python\\python37-32\\lib\\json\\encoder.py\", line 179, in default\n    raise TypeError(f'Object of type {o.__class__.__name__} '\nTypeError: Object of type TextIOWrapper is not JSON serializable\n"
            }
        }
    ],
    "errors": [
        {
            "message": "TypeError: Object of type TextIOWrapper is not JSON serializable",
            "description": "Traceback (most recent call last):\n  File \"c:\\users\\alexandr.may\\appdata\\local\\programs\\python\\python37-32\\lib\\site-packages\\golem\\test_runner\\test_runner.py\", line 173, in run_test\n    self.test_module.test(execution.data)\n  File \"C:\\Users\\alexandr.may\\testdir\\projects\\some_project\\tests\\second_test.py\", line 32, in test\n    json.dump(file, data)\n  File \"c:\\users\\alexandr.may\\appdata\\local\\programs\\python\\python37-32\\lib\\json\\__init__.py\", line 179, in dump\n    for chunk in iterable:\n  File \"c:\\users\\alexandr.may\\appdata\\local\\programs\\python\\python37-32\\lib\\json\\encoder.py\", line 438, in _iterencode\n    o = _default(o)\n  File \"c:\\users\\alexandr.may\\appdata\\local\\programs\\python\\python37-32\\lib\\json\\encoder.py\", line 179, in default\n    raise TypeError(f'Object of type {o.__class__.__name__} '\nTypeError: Object of type TextIOWrapper is not JSON serializable\n"
        }
    ],
    "description": "",
    "browser": "chrome (remote)",
    "test_data": {
        "order_type": "'Plats'",
        "document_type": "'CORRECTION OF PLAT (PL)'",
        "num_of_pages": "'1'",
        "num_of_lots": "'1'",
        "section": "'GRANTOR'",
        "first_name": "'First Name'",
        "alert_text": "'Please Enter Account OR Email OR Name'",
        "env": "{'url': 'http://{}:{}@10.0.1.73:8001/69999/?ReturnUrl=https://crs.qa.kofile.com/69999/Account/ReturnClaim&amp;Referrer=', 'login': 'User1', 'password': 'Automation_1_3', 'getItemsLink': 'https://crs.qa.kofile.com/69999/api/Order/GetQueuesItemCounts', 'cookieName': 'ASP.NET_SessionId', 'name': 'test_env'}"
    },
    "environment": "test_env",
    "set_name": "Plats",
    "test_elapsed_time": 0.89,
    "test_timestamp": "2019.11.01.11.40.48.731"
}